src/i18n/fi.json

```diff
@@ -122,7 +122,7 @@
     "ruotsi": "ruotsi",
     "suomi": "suomi",
 
-    "validation-isUrl": "Kirjoita URL osoite kokonaisena ja oikeassa muodossa (http://...)",
+    "validation-isUrl": "Tarkista URL osoitteen kirjoitusmuoto",
```

Comment:
Ah, noticed a typo in the old validation text, it should of course be `URL-osoitteen` :D